visualime/explain.py

```diff
@@ -147,55 +147,63 @@ def render_explanation(
     positive: Optional[Union[Tuple[int, int, int], str]] = "green",
     negative: Optional[Union[Tuple[int, int, int], str]] = None,
     opacity: float = 0.7,
-    coverage: float = 0.2,
+    coverage: Optional[float] = None,
+    num_of_segments: Optional[int] = None,
```

Comment:
`coverage` sollte einen Default-Wert haben, so dass die Funktion in jedem Fall läuft, wenn man nur `image`, `segment_mask` und `segment_weights` angibt.